src/app/components/VisuallyHiddenText/index.test.tsx

```diff
@@ -9,4 +9,10 @@ describe('VisuallyHiddenText', () => {
     );
     expect(container.firstChild).toMatchSnapshot();
   });
+  it('should render off screen text for screen readers as an h1', () => {
```

Comment:
*teeny tiny nitpick*: whitespace
```suggestion
  
  it('should render off screen text for screen readers as an h1', () => {
```